frontend/src/components/Tools/Tools.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Container, Row, Col } from 'reactstrap';
+import "./Tools.css";
+
+// Icons
+import { ReactComponent as ReactIcon } from './res/react-icon.svg';
+import { ReactComponent as NodeIcon } from './res/node-icon.svg';
+import { ReactComponent as JiraIcon } from './res/jira-icon.svg';
+import { ReactComponent as PostmanIcon } from './res/postman-icon.svg';
+import { ReactComponent as AWSIcon } from './res/aws-icon.svg';
+import { ReactComponent as HerokuIcon } from './res/heroku-icon.svg';
```

Comment:
What do you think about putting the images in the public folder and importing them from there instead. I think it would be nice to only have components inside of the components folder and not other things like images and styles.

Reply:
I had some trouble implementing this due to the nature of React. Apparently, relative imports outside of the `src` folder are not supported. Would it be okay if I instead created an images folder within the `src` directory?

Reply:
Oh right ok. Images folder sounds good.